frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -70,93 +73,133 @@ export default function PersonalInformationIndex() {
-                {t('personal-information:index.change-preferred-language')}
-              </InlineLink>
-            </p>
-          )}
-        </DescriptionListItem>
-        <DescriptionListItem term={t('personal-information:index.home-address')}>
-          {personalInformation.homeAddress ? (
-            <Address
-              address={personalInformation.homeAddress.streetName ?? ''}
-              city={personalInformation.homeAddress.cityName ?? ''}
-              provinceState={regionList.find((region) => region.provinceTerritoryStateId === personalInformation.homeAddress!.provinceTerritoryStateId)?.abbr}
-              postalZipCode={personalInformation.homeAddress.postalCode}
-              country={countryList.find((country) => country.countryId === personalInformation.homeAddress!.countryId)?.[i18n.language === 'fr' ? 'nameFr' : 'nameEn'] ?? ' '}
-            />
-          ) : (
-            <p>{t('personal-information:index.no-address-on-file')}</p>
-          )}
-          {useFeature('edit-personal-info') && (
-            <p>
-              <InlineLink id="change-home-address-button" routeId="$lang+/_protected+/personal-information+/home-address+/edit" params={params}>
-                {t('personal-information:index.change-home-address')}
-              </InlineLink>
-            </p>
-          )}
-        </DescriptionListItem>
-        <DescriptionListItem term={t('personal-information:index.mailing-address')}>
-          {personalInformation.mailingAddress ? (
-            <Address
-              address={personalInformation.mailingAddress.streetName ?? ''}
-              city={personalInformation.mailingAddress.cityName ?? ''}
-              provinceState={regionList.find((region) => region.provinceTerritoryStateId === personalInformation.mailingAddress!.provinceTerritoryStateId)?.abbr}
-              postalZipCode={personalInformation.mailingAddress.postalCode}
-              country={countryList.find((country) => country.countryId === personalInformation.mailingAddress!.countryId)?.[i18n.language === 'fr' ? 'nameFr' : 'nameEn'] ?? ''}
-            />
-          ) : (
-            <p>{t('personal-information:index.no-address-on-file')}</p>
-          )}
-          {useFeature('edit-personal-info') && (
-            <p>
-              <InlineLink id="change-mailing-address-button" routeId="$lang+/_protected+/personal-information+/mailing-address+/edit" params={params}>
-                {t('personal-information:index.change-mailing-address')}
-              </InlineLink>
-            </p>
-          )}
-        </DescriptionListItem>
-        <DescriptionListItem term={t('personal-information:index.phone-number')}>
+        <DescriptionListItemTwoColumn term={t('personal-information:index.addresses')}>
+          <dl>
+            <dt className="text-lg font-semibold">{t('personal-information:index.home-address')}</dt>
```

Comment:
Might just be me, but I feel that these headings are too large as they are larger than `<h1>` page title. I feel they should be the same size or smaller than the `<h1>` page title.

Reply:
What browser are you using?

In Firefox these headings and page title are the same size, that is 1.875 rem.

Reply:
I'm using Firefox as well and you are right, the `font-size` for both are the same but because of the different `font-family`, the headings appear larger. 

In the below screenshot, "Full Name" uses `Noto Sans` and "Personal Information" uses `Lato`:

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/bad0447e-1063-4226-aa30-7dd3c79f80c7)

Reply:
Ah I'd missed that.

As of commit f47f3a28ff31638fd1ad565d97b2ba6787bdc49a they use the same font.